tutorials/chapter1.md

```diff
@@ -57,19 +58,27 @@ There are two important elements in this class that are always required when pro
 * The ```publish``` method expects a const reference to the publishing object. All publisher implementations make a copy of this object before proceeding to the actual publication. There is another API for publication that skip the copy, that we will cover later.
 * Application code always use pointers to ```kpsr::Publisher```. This is done in order to keep the application independent of the implementation of the publisher. It is not only a good practice, but also necessary when building unit tests for classes like this example. The same principle applies to the ```kpsr::Subscriber``` and ```kpsr::Environment```.
 
+It is also important to note that the IDs of the different threads that are running during the execution of each example are printed out in order to clarify the number of threads involved in each example. Then, this is de purpose of the line where the thread ID is printed out in class SimplePublisher.
```

Comment:
Instead of this text:
"_Then, this is de purpose of the line where the thread ID is printed out in class SimplePublisher._"
Please explain all 5 threads and what are they doing:
- Main thread: blah
- DataMultiplexer Consumer 1 (Mod): Blah
- etc.

Reply:
Done